src/app/ui/learn/LearnPage.tsx

```
@@ -1,25 +1,24 @@
 import Box from '@mui/material/Box';
+import { LearnPageBlogCarousel } from './LearnPageBlogCarousel';
+import { LearnFilteringProvider } from '../../../providers/LearnProvider/filtering/LearnFilteringContext';
+import { LearnPageArticlesSection } from './LearnPageArticlesSection/LearnPageArticlesSection';
+import { JoinDiscordBanner } from '@/components/JoinDiscordBanner/JoinDiscordBanner';
+import { getFeaturedArticle } from '@/app/lib/getFeaturedArticle';
+import { getArticles } from '@/app/lib/getArticles';
+import { getTags } from '@/app/lib/getTags';
+import { Suspense } from 'react';
+import { LearnPageArticlesSectionSkeleton } from './LearnPageArticlesSection/LearnPageArticlesSectionSkeleton';
+
+interface LearnPageProps {}
```

Biome (2.4.7)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)